defaults/defaults.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for defaults 1.0.3
+// Project: https://github.com/tmpvar/defaults/
+// Definitions by: Ibtihel CHNAB <https://github.com/IbtihelCHNAB/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function defaults(options: any, defaultOptions: any): any;
+ declare module "defaults" {
+   export = defaults;
```

Comment:
Please use 4 spaces or 1 tab to indent, and also do not indent the lines above and below this.